example-library/src/main/resources/application-example-perf-k8s.yaml

```diff
@@ -0,0 +1,1297 @@
+  format: avro
+
+population:
+  userProvider: example
+  users:
+  - userId: Alice
+    auths:
+      public
+      private
+    roles:
+      HR
+    trainingCourses:
+      PAYROLL_TRAINING_COURSE
+
+  policyProvider: std
+  policies:
+  - resource: ${default.rootId}/with-policy/
+    owner: Alice
+    recordRules:
+      "1-Bank details only visible for the purpose of salary": uk.gov.gchq.palisade.example.library.rule.BankDetailsRule
+      "2-Emergency numbers only available for duty of care purpose": uk.gov.gchq.palisade.example.library.rule.DutyOfCareRule
+      "3-Nationality is only available for staff report": uk.gov.gchq.palisade.example.library.rule.NationalityRule
+      "4-Address masked for estates staff and otherwise only available for duty of care": uk.gov.gchq.palisade.example.library.rule.ZipCodeMaskingRule
+      "5-Record content masked for all who are not in the employee's management chain or part of the estates or HR.": uk.gov.gchq.palisade.example.library.rule.RecordMaskingRule
+    resourceRules:
+      "1-Only HR can access the first resource": uk.gov.gchq.palisade.example.library.rule.FirstResourceRule
+  - resource: ${default.rootId}/no-policy/
+    owner: Alice
+    resourceRules:
+      "1-Pass through rule for no-policy directory": uk.gov.gchq.palisade.example.library.rule.PassThroughRule
+    recordRules: {}
+
+  resourceProvider: std
+  resources:
+  - resourceId: ${default.rootId}/no-policy/employee_large.avro
+    rootId: ${default.rootId}/
+    connectionDetail: ${default.connectionDetail}
+    attributes:
+      type: ${default.type}
+      serialisedFormat: ${default.format}
+  - resourceId: ${default.rootId}/no-policy/employee_small.avro
+    rootId: ${default.rootId}/
+    connectionDetail: ${default.connectionDetail}
+    attributes:
+      type: ${default.type}
+      serialisedFormat: ${default.format}
+  - resourceId: ${default.rootId}/with-policy/employee_large.avro
+    rootId: ${default.rootId}/
+    connectionDetail: ${default.connectionDetail}
+    attributes:
+      type: ${default.type}
+      serialisedFormat: ${default.format}
+  - resourceId: ${default.rootId}/with-policy/employee_small.avro
+    rootId: ${default.rootId}/
+    connectionDetail: ${default.connectionDetail}
+    attributes:
+      type: ${default.type}
+      serialisedFormat: ${default.format}
+    # if you want more resources, macros are your friend here
+  - resourceId: ${default.rootId}/no-policy/many-employees/duplicate_000000/employee_000000.avro
+    rootId: ${default.rootId}/
+    connectionDetail: ${default.connectionDetail}
+    attributes:
+      type: ${default.type}
+      serialisedFormat: ${default.format}
+  - resourceId: ${default.rootId}/no-policy/many-employees/duplicate_000000/employee_000000.avro
+    rootId: ${default.rootId}/
+    connectionDetail: ${default.connectionDetail}
+    attributes:
+      type: ${default.type}
+      serialisedFormat: ${default.format}
```

Comment:
One of these duplicates needs be removed.

Reply:
Good spot